src/pages/library/Datasets.js

```diff
@@ -244,35 +242,29 @@ const amppd = () =>
     ]
   );
 
-const axin = () =>
+const cohortBuilder = () =>
   h(
     Participant,
     {
-      logo: { src: axinLogo, alt: 'AnalytiXIN logo', height: '55%' },
-      title: 'AnalytiX Indiana Health Dataset',
+      logo: { src: broadLogo, alt: 'Broad logo', height: '40%' },
+      title: 'Cohort Builder Demo',
       description: h(Fragment, [
-        'The ',
-        h(Link, { href: 'https://analytixindiana.com', ...Utils.newTabLinkProps }, 'AnalytiXIN life sciences health data asset (HDA)'),
-        ` has been designed through a working
-        collaboration involving Eli Lilly and Company, IU Health, the Indiana Biobank within the
-        IU School of Medicine, the Indiana Health Information Exchange, and other partners to build
-        a shared health-data platform linking consented clinical and genomic patient data. This
-        collaboration is further leveraging the Broad Institute’s technical and genomic
-        infrastructure to accelerate the build-out as well as connect to broader national
-        initiatives such as the National Institutes of Health All of Us research program.`,
+        'The cohort builder is an alpha feature in Terra, currently being designed and build for use to help with subsetting large datasets. This card takes you to a demo dataset for use in demoing and exploring the alpha cohort builder.',
       ]),
       sizeText: 'Participants: > 35,000',
     },
     [
       h(
         ButtonPrimary,
         {
-          'aria-label': 'View AnalytiXIN details',
+          'aria-label': 'View Cohort Builder Demo',
           tooltip: 'View details about this dataset',
-          href: Nav.getLink('dataset-builder-details', { snapshotId: brands.analytixin.datasetBuilderSnapshotId }),
-          onClick: () => captureBrowseDataEvent('AXIN'),
+          href: Nav.getLink('dataset-builder-details', {
+            snapshotId: getConfig().terraDeploymentEnv === 'dev' ? 'c3eb4708-444f-4cbf-a32c-0d3bb93d4819' : '1f2c98d2-ccec-410e-8242-d995f9aa2ec6',
```

Comment:
I think these are the Azure datasets. Doesn't necessarily need to happen in this PR, but it could be a good time to just change these over to the GCP ones.